core/src/integTest/scala/ai/h2o/sparkling/IntegrationTestSuite.scala

```diff
@@ -0,0 +1,153 @@
+/*
+* Licensed to the Apache Software Foundation (ASF) under one or more
+* contributor license agreements.  See the NOTICE file distributed with
+* this work for additional information regarding copyright ownership.
+* The ASF licenses this file to You under the Apache License, Version 2.0
+* (the "License"); you may not use this file except in compliance with
+* the License.  You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+package ai.h2o.sparkling
+
+import ai.h2o.sparkling.ml.utils.{ComplexSchema, SchemaUtils}
+import org.apache.spark.SparkContext
+import org.apache.spark.h2o.utils.{SharedH2OTestContext, TestFrameUtils}
+import org.apache.spark.h2o.{DoubleHolder, H2OConf, H2OContext}
+import org.apache.spark.sql.Row
+import org.junit.runner.RunWith
+import org.scalatest.FunSuite
+import org.scalatest.junit.JUnitRunner
+
+import scala.concurrent.duration.Duration
+
+@RunWith(classOf[JUnitRunner])
+class IntegrationTestSuite extends FunSuite with SharedH2OTestContext {
+
+  override def createSparkContext: SparkContext = new SparkContext("local-cluster[2,1,2024]", this.getClass.getName, conf = defaultSparkConf)
+
+  test("verify H2O cloud building on local cluster") {
+    val hc = H2OContext.getOrCreate(new H2OConf().setClusterSize(1))
+    if (hc.getConf.runsInInternalClusterMode) {
+      assert(hc.getH2ONodes().length == 2)
+    } else {
+      assert(hc.getH2ONodes().length == 1)
+    }
+  }
+
+  test("Convert H2OFrame to DataFrame when H2OFrame was changed in DKV") {
+    val rdd = sc.parallelize(1 to 100, 2)
+    val h2oFrame = hc.asH2OFrame(rdd)
+    assert(h2oFrame.anyVec().nChunks() == 2)
+    val updatedFrame = h2oFrame.add(h2oFrame)
+
+    val convertedDf = hc.asDataFrame(updatedFrame)
+    convertedDf.collect()
+
+    assert(convertedDf.count() == h2oFrame.numRows())
+    assert(convertedDf.columns.length == h2oFrame.names().length)
+  }
+
+  test("Task killed but frame still converted successfully") {
```

Comment:
The tests were moved from different test suites with names identifying the scope of testing. Shouldn't the test names also contain information defined by the former suite names? ("H2OFrameToDataFrameHA")